.stylelintrc

```diff
@@ -1,39 +1,21 @@
 {
-  "extends": "stylelint-config-recommended-scss",
+  "extends": "stylelint-config-standard",
   "ignoreFiles": [
     "**/Public/**",
     "**/Private/Templates/**",
     "node_modules/**",
     "**/*.noLinter.*"
   ],
   "rules": {
-    "property-no-unknown": [
-      true,
-      {
-        "ignoreProperties": [
-          "font-range",
-          "min-font-size",
-          "max-font-size",
-          "lower-font-range",
-          "upper-font-range",
-          "font-awesome",
-          "tap-highlight-color",
-          "touch-callout"
-        ]
-      }
-    ],
-    "selector-pseudo-class-no-unknown": [
-      true,
-      {
-        "ignorePseudoClasses": ["at-least", "at-most", "between", "exactly"]
-      }
+    "selector-class-pattern": "[a-z-]+",
```

Comment:
Why this? I sometimes need to use numbers, underscores and sometimes need to augment certain tailwind stuff.